mne/digitization/_utils.py

```diff
@@ -0,0 +1,393 @@
+                                hpi=hpi,
+                                extra_points=extra)
+
+    return info_dig, ctf_head_t
+
+
+##############################################################################
+# From mne.io.kit
+def _set_dig_kit(mrk, elp, hsp):
+    """Add landmark points and head shape data to the KIT instance.
+
+    Digitizer data (elp and hsp) are represented in [mm] in the Polhemus
+    ALS coordinate system. This is converted to [m].
+
+    Parameters
+    ----------
+    mrk : None | str | array_like, shape (5, 3)
+        Marker points representing the location of the marker coils with
+        respect to the MEG Sensors, or path to a marker file.
+    elp : None | str | array_like, shape (8, 3)
+        Digitizer points representing the location of the fiducials and the
+        marker coils with respect to the digitized head shape, or path to a
+        file containing these points.
+    hsp : None | str | array, shape (n_points, 3)
+        Digitizer head shape points, or path to head shape file. If more
+        than 10`000 points are in the head shape, they are automatically
+        decimated.
+
+    Returns
+    -------
+    dig_points : list
+        List of digitizer points for info['dig'].
+    dev_head_t : dict
+        A dictionary describe the device-head transformation.
+    """
+    from ..coreg import fit_matched_points, _decimate_points
+    from ..io.kit.constants import KIT
+    from ..io.kit.coreg import read_mrk
+
+    if isinstance(hsp, str):
+        hsp = _read_dig_points(hsp)
+    n_pts = len(hsp)
+    if n_pts > KIT.DIG_POINTS:
+        hsp = _decimate_points(hsp, res=0.005)
+        n_new = len(hsp)
+        warn("The selected head shape contained {n_in} points, which is "
+             "more than recommended ({n_rec}), and was automatically "
+             "downsampled to {n_new} points. The preferred way to "
+             "downsample is using FastScan.".format(
+                 n_in=n_pts, n_rec=KIT.DIG_POINTS, n_new=n_new))
+
+    if isinstance(elp, str):
+        elp_points = _read_dig_points(elp)
+        if len(elp_points) != 8:
+            raise ValueError("File %r should contain 8 points; got shape "
+                             "%s." % (elp, elp_points.shape))
+        elp = elp_points
+    elif len(elp) != 8:
+        raise ValueError("ELP should contain 8 points; got shape "
+                         "%s." % (elp.shape,))
+    if isinstance(mrk, str):
+        mrk = read_mrk(mrk)
+
+    mrk = apply_trans(als_ras_trans, mrk)
+
+    nasion, lpa, rpa = elp[:3]
+    nmtrans = get_ras_to_neuromag_trans(nasion, lpa, rpa)
+    elp = apply_trans(nmtrans, elp)
+    hsp = apply_trans(nmtrans, hsp)
+
+    # device head transform
+    trans = fit_matched_points(tgt_pts=elp[3:], src_pts=mrk, out='trans')
+
+    nasion, lpa, rpa = elp[:3]
+    elp = elp[3:]
+
+    dig_points = _make_dig_points(nasion, lpa, rpa, elp, hsp)
+    dev_head_t = Transform('meg', 'head', trans)
```

Comment:
To be consistent with previous call:
`dev_head_t = Transform(fro='meg', to='head', trans=trans)`

Reply:
agreed. I'll at them on the next PR.